recipes/ccc/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "ccc" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PSLmodels/Cost-of-Capital-Calculator/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ad72b2a2570e5dae9c122f31ab8113b1ef2a36caf8029a9ec3cda1a0d0eaea5d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    - taxcalc >=3.1.0
+    - bokeh >=1.4.0
+    - paramtools >=0.7.1
+    - setuptools
```

Comment:
It looks like `setuptools` is only needed to run `setup.py`, so I *think* you can move this up under `host` instead.

Reply:
Good call - moved.